### PR TITLE
Fixed forward bearing computation.

### PR DESCRIPTION
**Pull request by ygorshenin**
Fixed calculation of forward bearing, reverse bearing calculations are still flawed :( Also, output of stats tool is ordered by segment id now, moreover, increased penalty for bearing mismatch.

### CODE REVIEW

**Comment by ygorshenin** [comment]
@mgsergio PTAL

File: openlr/openlr_simple_decoder.cpp [hunk 1]
```diff
@@ -431,26 +443,34 @@ class AStarRouter
...
     size_t m_stage = 0;
+    bool m_bearingChecked = false;
   };
 
   using Links = map<Vertex, pair<Vertex, Edge>>;
 
   class Score
   {
   public:
+    static const int kFakeCoeff = 3;
```
Comment (mgsergio) [inline openlr/openlr_simple_decoder.cpp:469]: s/Coeff/Factor

File: openlr/openlr_simple_decoder.cpp [hunk 2]
```diff
@@ -286,7 +286,7 @@ class AStarRouter
       }
     };
 
-    Vertex const s(js, js, 0, 0 /* stage */);
+    Vertex const s(js, js, 0, 0 /* stage */, false /* bearingChecked */);
```
Comment (mgsergio) [inline openlr/openlr_simple_decoder.cpp]: /* stageStartDistance */

File: openlr/openlr_simple_decoder.cpp (hunk 1 above)
Comment (ygorshenin) [reply openlr/openlr_simple_decoder.cpp:469]: Не, я бы оставил Coeff. Factor это скорее делитель, нежели чем домножающий коэффициэнт.

File: openlr/openlr_simple_decoder.cpp (hunk 2 above)
Comment (ygorshenin) [reply openlr/openlr_simple_decoder.cpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Pull request merged**